SECloseVoteRequestGenerator.user.js

```diff
@@ -1493,6 +1501,8 @@
         var item = this.item;
         this.requestReasonInput = $('input[type="text"]', item);
         var requestTypeInput = this.requestTypeInput = $('select[name="requestType"]', item);
+        var SDReport = this.SDReport = $('#chkSDReport', item);
+        var NATO = this.NATO = $('#chkNATO', item);
```

Comment:
Please use variable names and class/object property names that are in camelCase. In general, starting a variable name with a capital letter tends to indicate that it's something that's being used as a `class`, or class-like `function`.

Reply:
In addition, it looks like you don't actually use the `SDReport` and `NATO` variables (you do use `this.SDReport` and `this.NATO`). Unless you use both versions, it's probably better to just create one or the other.

Within this section of the code, the `this.*` definitions exist because the information they contain is needed in one of methods defined for the `GuiItemSend` class-like function. For what your using them for, I'd probably use the `var` variables.

I'd probably also call the variables something that's a little more descriptive of what they contain. For example, I'd probably use `sdReportCheckbox` and `natoCheckbox`.

[Obviously, you'll need to change the selectors to not use the `id` attributes.]